pjlib/include/pj/compat/socket.h

```diff
@@ -61,7 +61,7 @@
  */
 #if defined(_MSC_VER) && defined(PJ_HAS_IPV6) && PJ_HAS_IPV6!=0
 #   ifndef s_addr
-#	define s_addr  S_un.S_addr
+#	define s_addr  S_un.addr
```

Comment:
this should not be changed?

Reply:
oh no, that's the evil of using batch replace. Need to recheck everything.